packages/hd-transport/src/types/messages.ts

```
@@ -2790,6 +2791,16 @@ export type DeviceEraseSector = {
   sector: number;
 };
 
+// UnLockDevice
+export type UnLockDevice = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)